_tpl/testPage.tpl

```diff
@@ -4,36 +4,6 @@ if (!empty($_POST)) {
 	include('_act/postComment.tpl');
 }
 
-if (in_array($item->browserscopeID, array(NULL, ''))) {
-	$item->browserscopeID = addBrowserscopeTest($item->title, $item->info, 'http://' . DOMAIN . '/' . $item->slug . ($item->revision > 1 ? '/' . $item->revision : ''));
-	$sql = 'UPDATE pages SET browserscopeID = "' . $db->real_escape_string($item->browserscopeID) . '" WHERE id = ' . $item->id . "\n";
-	$db->query($sql);
-}
-
-// Don’t let robots index non-published test cases
-if ($item->visible === 'n' && (isset($_SESSION['own'][$item->id]) || isset($_SESSION['admin']))) {
-	$noIndex = true;
-}
-
-$benchmark = $showAtom = $jsClass = true; require('tpl-inc/head.tpl'); ?>
-<hgroup>
-	<h1><?php echo addCode(he($item->title)); ?></h1>
-	<h2>JavaScript performance comparison</h2>
-</hgroup>
-<p class="meta"><?php if ($item->revision > 1) { ?>Revision <?php echo $item->revision; ?> of this test case<?php } else { ?>Test case<?php } ?> created <?php echo author($item->author, $item->authorURL); ?><time datetime="<?php echo date('c', strtotime($item->published)); ?>" pubdate><?php echo relativeDate($item->published); ?></time><?php if (relativeDate($item->published) !== relativeDate($item->updated)) { ?> and last updated <time datetime="<?php echo date('c', strtotime($item->updated)); ?>"><?php echo relativeDate($item->updated); ?></time><?php } if ($item->visible === 'n' && (isset($_SESSION['own'][$item->id]) || isset($_SESSION['admin']))) { ?> <strong><a href="/<?php echo $slug . ($item->revision > 1 ? '/' . $item->revision : '') . '/publish'; ?>">Not published yet!</a></strong> · <a href="/<?php echo $slug . ($item->revision > 1 ? '/' . $item->revision : '') . '/edit'; ?>">Edit</a><?php } ?></p>
-<?php if ($item->info) { ?>
-<section>
-<h2>Info</h2>
-<?php
-	echo md($item->info);
-?>
-</section>
-<?php } ?>
-<?php if ($item->initHTML || $item->setup || $item->teardown) { ?>
-<section id="prep-code">
-<h2>Preparation code</h2>
-<pre><code><?php
-
 class Swap {
 	public static $items = array();
 
```

Comment:
@mathiasbynens I can't find `$scripts` defined anywhere for lines 26 and 29 below. safe to delete?